recipes/icu/all/conandata.yml

```diff
@@ -43,3 +43,5 @@ patches:
       base_path: "source_subfolder"
     - patch_file: "patches/0001-71.1-fix-undef-strict-ansi.patch"
       base_path: "source_subfolder"
+    - patch_file: "patches/0001-71.1-fix-emscripten.patch"
```

Comment:
Can you please add the patch fields for this new one?
https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches-fields

The type and description would be amazing.


I would also suggest opening a PR again upstream so they can improve their build scripts 💪 then you can add it as a `patch_source` too